langserver/handlers/complete_test.go

```diff
@@ -84,19 +84,22 @@ func TestCompletion_withValidData(t *testing.T) {
 					{
 						"label":"anonymous",
 						"kind":5,
-						"detail":"(Optional, number) Desc 1",
+						"detail":"Optional, number",
+						"documentation":"Desc 1",
 						"insertTextFormat":1
 					},
 					{
 						"label":"base_url",
 						"kind":5,
-						"detail":"(Optional, string) Desc 2",
+						"detail":"Optional, string",
+						"documentation":"Desc 2",
 						"insertTextFormat":1
 					},
 					{
 						"label":"individual",
 						"kind":5,
-						"detail":"(Optional, bool) Desc 3",
+						"detail":"Optional, bool",
+						"documentation":"Desc 3",
```

Comment:
The `testSchemaOutput` (mocking output from `providers schema -json`) will also need updating to reflect the addition of `documentation` field.

I'm assuming it's OK to _just_ update it as both `terraform-json` & protocol changes are additive and the missing fields would just become empty strings and cause no harm anywhere? i.e. users with older version of Terraform prior to https://github.com/hashicorp/terraform/pull/22583 still get a decent UX?

Reply:
Yeah, its all additive in the JSON